### PR TITLE
Polish the tutorial

### PR DESCRIPTION
**Pull request by milan-sedivy**
Creating this as a draft for now, I think it will need a few iterations from other people. 

Changes **(so far)**: 
- Added more fluff to make it beginner friendly
- Added tabgroups to distinguish between npm/yarn/pnpm (you can now select one anywhere and it changes across the tutorial pages)
- Polished up some parts by adding additional information that was missing and would lead the reader to encounter errors 
- Added 2 pictures for hints

### CODE REVIEW

File: docs/tutorial/Basic/Setting Up.mdx [hunk 1]
```diff
@@ -75,10 +76,45 @@ Change directory into the repository and install the project dependencies
...
+# default wallet of localfhenix
+export MNEMONIC="demand hotel mass rally sphere tiger measure sick spoon evoke fashion comfort"
+export WALLET=""
+```
+#### Starting LocalFhenix.
+
+Use the following command:
+
+<Tabs groupId="package-managers">
+    <TabItem value="npm" label="npm">
```
Comment (toml01) [inline docs/tutorial/Basic/Setting Up.mdx:102]: Have you actually tried running these? I recall it used to not work with package managers other than pnpm for some reason, not sure if that's still the case or not

**Review by toml01 — COMMENTED** [review]
Overall looks good!

File: docs/tutorial/Basic/Setting Up.mdx (hunk 1 above)
Comment (milan-sedivy) [reply docs/tutorial/Basic/Setting Up.mdx:102]: @toml01 so I did try running them, just not on a properly clean repo. Everything seemed to work fine. Just as a sanity-check I might pull 3 different clones and test it out again.

File: docs/tutorial/Basic/Setting Up.mdx [hunk 2]
```diff
@@ -75,10 +76,45 @@ Change directory into the repository and install the project dependencies
 
 Alternative set up paths can be just cloning or forking the example repository, or starting from scratch using our [fhenix hardhat plugin](../../devdocs/docs/hardhat), [Gitpod environment](../../devdocs/docs/gitpod) or Github Workspaces
 
-## Starting Localfhenix
+## Configuring and Starting Localfhenix
+In order for us to locally deploy and test contracts we will need to run our own node, for this specific use-case we have a docker image named `localfhenix`.
+For ease of use and a simple setup we have hardhat task `localfhenix:start` that downloads and runs the image in a container for you.
```
Comment (Cashmaney) [inline docs/tutorial/Basic/Setting Up.mdx:81]: We should probably have a tip or something here that you need docker installed for this to work

**Comment by Cashmaney** [comment]
Looks good so far 👍

File: docs/tutorial/Basic/Setting Up.mdx (hunk 2 above)
Comment (milan-sedivy) [reply docs/tutorial/Basic/Setting Up.mdx:81]: Will add it to the pre-requisites with a link to their docs

File: docs/tutorial/Basic/Setting Up.mdx (hunk 1 above)
Comment (milan-sedivy) [reply docs/tutorial/Basic/Setting Up.mdx:102]: So I did test it today, and it does work. However I've looked into how the package managers work and I will probably add a Disclaimer on top saying that our projects mainly use (p)npm and that although yarn can be used it is not recommended. (Due to missing lock files)

**Pull request merged**